ChangeLog

```diff
@@ -53,6 +53,7 @@ Features
      https://sweet32.info/SWEET32_CCS16.pdf.
    * Extend the  MBEDTLS_SSL_EXPORT_KEYS to export the handshake randbytes,
      and the used tls-prf.
+   * Add public API for tls-prf function, according to requested enum.
```

Comment:
This section belongs to the already released 2.17.0 branch, we should add this entry to the "Features"  section of the current development branch. Also, we should add it to the "API changes" section as well.

Reply:
my bad...

Reply:
Why is it an API change?

Reply:
New public function in the API. (Here API means not a single function signature as people sometimes use it, but in the general sense where API means the set of all the public structures, constants functions and macros together.)